certora/specs/Reentrancy.spec

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+methods {
+    function multicall(bytes[]) external returns(bytes[]) => NONDET DELETE;
+
+    function _.accrueInterest(MetaMorphoHarness.MarketParams) external => voidSummary() expect void;
+    function _.supply(MetaMorphoHarness.MarketParams, uint256, uint256, address, bytes) external => uintPairSummary() expect (uint256, uint256);
+    function _.withdraw(MetaMorphoHarness.MarketParams, uint256, uint256, address, address) external => uintPairSummary() expect (uint256, uint256);
+
+    function _.transfer(address, uint256) external => boolSummary() expect bool;
+    function _.transferFrom(address, address, uint256) external => boolSummary() expect bool;
+    function _.balanceOf(address) external => uintSummary() expect uint256;
```

Comment:
would it be worth having comments here saying what we expect from external contracts (erc20 isn't reentrant, morpho isn't reentrant)?

Reply:
actually it's written later
```
// Ignore calls to tokens and Morpho markets as they are trusted (they have gone through a timelock).
```